selfdrive/camerad/cameras/camera_qcom2.c

```diff
@@ -484,10 +484,10 @@ void enqueue_buffer(struct CameraState *s, int i) {
   if (s->buf_handle[i]) {
```

Comment:
Is there no better source for the `request_id` here? Seems kinda hacky to get it from the metadata...